### DISCUSSION TITLE
Negative stress predictions vs. expected?

### DISCUSSION THREAD

**Discussion by nwong70** [ACEsuit/mace#1078] [Q&A] [answered]
Hello,

I have performed a fine-tuning step on my liquid system of about 200 atoms with about 30 different structures.  I am able to get RMSE_E and RMSE_F down to reasonable values, and stress seems to fail to train.  However, when I look at my parity plot for stress, it shows that my predictions are about negative of what DFT outputs. Is this to be expected? My stress training data is pulled straight from VASP using pymatgen and divided by ~1602 to get from GPa to eV/A^3.  The pretrained head does not have this issue.

My configs, log, and finetuning dataset are below.  

> python -m mace.cli.run_train \
  --name="finetuned_model" \
  --pt_train_file="./universal_to_ft1_selected_data.xyz" \
  --train_file="./universal_to_ft1.xyz" \
  --valid_fraction=0.1 \
  --enable_cueq=True \
  --foundation_model_elements=True \
  --foundation_model="/models/mace_agnesi_small.model" \
  --foundation_head="pt_head" \
  --filter_type_pt="none" \
  --error_table="PerAtomRMSE" \
  --batch_size=32 \
  --max_num_epochs=1 \
  --forces_weight=550 \
  --energy_weight=10000 \
  --swa \
  --swa_forces_weight=550 \
  --swa_energy_weight=10000 \
  --swa_stress_weight=5 \
  --swa_lr=0.00025 \
  --ema \
  --ema_decay=0.99 \
  --amsgrad \
  --device=cuda \
  --seed=1
[finetuning.log.txt](https://github.com/user-attachments/files/21237559/finetuning.log.txt)
[universal_to_ft1.xyz.txt](https://github.com/user-attachments/files/21237578/universal_to_ft1.xyz.txt)

<img width="3034" height="1834" alt="ft1_to_ft2_run-1_train_Default_stage_two" src="https://github.com/user-attachments/assets/0db400f8-2ac8-4855-93aa-0b2f0bc8403e" />

<img width="3034" height="1834" alt="ft1_to_ft2_run-1_train_pt_head_stage_two" src="https://github.com/user-attachments/assets/26333fce-145b-46da-8bed-7f8262d5b9cd" />

**Comment by gabor1** [comment, marked as answer]
vasp and ASE define the stress with opposite sign. So flip the sign on your data

**Reply by nwong70** [reply]
oh excellent, thanks
